OpenImis.ePayment/Data/ImisCoverage.cs

```diff
@@ -25,25 +25,8 @@ public ImisCoverage(IConfiguration configuration)
             SqlParameter[] sqlParameters = {
                 new SqlParameter("@InsureeNumber", insureeNumber),
             };
-            DataTable response = new DataTable();
-
-            try
```

Comment:
Why the Try/Catch block is removed? Can't we log the error rather than removing it?

Reply:
Because this block just rethrowed the sql exception and replaced any other exception with empty generic exception, crashing the endpoint and hiding the cause. Without it the escaped exception will be caught and logged